lib/cartopy/tests/mpl/test_ticks.py

```diff
@@ -82,7 +84,9 @@ def test_set_xticks_non_cylindrical():
     plt.close()
 
 
-@ImageTesting(['yticks_no_transform'], tolerance=0.125)
+@ImageTesting(['yticks_no_transform'],
+              tolerance={'1.3': 8.5, '1.4': 8.5,  # Text bounding box changes.
```

Comment:
I've taken a look at this image, and it highlights the need for us to do something more sophisticated in our tests.
In order to avoid a high RMS, I'm considering having two separate result images for this one.